content-src/components/Hint/Hint.scss

```diff
@@ -0,0 +1,90 @@
+.hint-container {
+  display: inline-block;
+  position: relative;
+  z-index: $hint-z-index;
+
+  // This is the link that if clicked, opens the explanation
+  .prompt {
+    color: inherit;
+    opacity: 0.8;
+
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+
+  // This is the box that shows up when you click the prompt
+  .explanation {
+    font-size: $hint-font-size;
+    box-shadow: 0 5px 10px $faintest-black;
+    line-height: $hint-line-height;
+    border: 1px solid $hint-border-color;
+    width: $hint-box-width;
+    border-radius: $hint-border-radius;
+    padding: $hint-padding;
+    background: $white;
+    position: absolute;
+    left: 100%;
+    margin-left: $hint-offset-left;
+    top: -$hint-offset-top;
+
+    &::before {
+      content: '';
+      position: absolute;
+      top: $hint-offset-top - 1;
+      left: -($hint-offset-left / 2);
+      width: $hint-offset-left;
+      height: $hint-offset-left;
+      background-color: $white;
+      display: block;
+      transform: rotate(45deg);
+      border: 1px solid $hint-border-color;
+      border-right-style: none;
+      border-top-style: none;
+    }
+
+    h3 {
+      color: $hint-title-color;
+      font-size: $hint-title-font-size;
+      line-height: 1.1;
+      font-weight: 500;
+      margin: 0 0 10px;
+    }
+
+    p {
+      color: $hint-body-color;
+      margin: 0 0 15px;
+    }
+
+    .close-button {
+      display: block;
```

Comment:
Does the above line have any effect?  Disabling it in dev-tools doesn't seem to make an obvious visual difference....